### PR TITLE
Add a simple notifier that prints stuff to the console.

### PR DESCRIPTION
**Pull request by kbruegge**
I think it's a good idea to have the most simple example one can think of. 
Also it can be used offline. Or without a phone. Or without an unimail thing.

### CODE REVIEW

File: custos/notify/console.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import logging
+
+from .base import Notifier
+
+log = logging.getLogger(__name__)
+
+
+class ConsoleNotifier(Notifier):
+    def notify(self, recipient, msg):
+        log.info('Message: {}    For: {}'.format(msg.text, recipient))
```
Comment (maxnoe) [inline custos/notify/console.py]: I think this should use the level of the `Message`, not just info?

File: custos/notify/console.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+        if msg.level == logging.INFO:
+            log.info('Message: {}    For: {}'.format(msg.text, recipient))
+        if msg.level == logging.DEBUG:
+            log.debug('Message: {}    For: {}'.format(msg.text, recipient))
+        if msg.level == logging.WARN:
+            log.warn('Message: {}    For: {}'.format(msg.text, recipient))
+        if msg.level == logging.CRITICAL:
+            log.critical('Message: {}    For: {}'.format(msg.text, recipient))
+        if msg.level == logging.ERROR:
+            log.error('Message: {}    For: {}'.format(msg.text, recipient))
```
Comment (maxnoe) [inline custos/notify/console.py]: Those level are only ints and people might define their own levels. You should maybe test for `<=` or `>=` in the proper order.

Comment (kbruegge) [reply custos/notify/console.py]: Maybe we should just `print` it here. For simplicities sake?

Comment (maxnoe) [reply custos/notify/console.py]: Yeah

**Comment by maxnoe** [comment]
Nice minimal example!

**Pull request merged**